assets/locales/en.json

```diff
@@ -13,7 +13,8 @@
   "openRepoDir": "Open Repository Directory",
   "openLogsDir": "Open Logs Directory",
   "openConfigFile": "Open Configuration File",
-  "viewOnGitHub": "View on GitHub",
+  "viewOnGitHub": "View on GitHub…",
+  "helpUsTranslate": "Help Translating…",
```

Comment:
“Help Translate This App...”?
I understand it’s going to expand the menu width, but it’s the shortest phrase I can find that makes it clear we are talking about human languages rather than programming languages.

Another option might be “IPFS Translation Project...”, but it doesn’t have a verb in there, which makes it inconsistent with other menu items.

Reply:
Uh, I prefer "Help Translate This App" on this case.

Reply:
I don't think it looks bad:

![image](https://user-images.githubusercontent.com/5447088/80283128-724a0680-870d-11ea-8303-497d9cd59ca8.png)

Reply:
Agree! Thanks.